frontend/pages/shopping-lists/_id.vue

```diff
@@ -56,14 +56,12 @@
 
       <!-- View By Label -->
       <div v-else>
-        <div v-for="(value, key, idx) in itemsByLabel" :key="key" class="mb-6">
+        <div v-for="(value, key) in itemsByLabel" :key="key" class="mb-6">
           <div @click="toggleShowChecked()">
-            <span v-if="idx || key !== $tc('shopping-list.no-label')">
-              <v-icon :color="getLabelColor(value[0])">
-                {{ $globals.icons.tags }}
-              </v-icon>
-            </span>
-            {{ key }}
+            <div class="text-left">
+              <BaseButton create :color="getLabelColor(value[0])" @click="createEditorOpen = true">{{ key }}</BaseButton>
```

Comment:
Lets have this be the color of the default label: 

```suggestion
            <BaseButton :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#757575'" @click="createEditorOpen = true">{{ key }}</BaseButton>
```